Examples/BookingSystem.AspNetFramework/Settings/EngineConfig.cs

```diff
@@ -166,6 +166,9 @@ public static StoreBookingEngine CreateStoreBookingEngine(AppSettings appSetting
                         ),
                     HasSingleSeller = appSettings.FeatureFlags.SingleSeller,
 
+                    // IdempotencyStore used for storing the response to Order Creation B/P requests
```

Comment:
This is a hangover from a previous PR (https://github.com/openactive/OpenActive.Server.NET/pull/207) where the Core code wasn't copied over to Framework